__tests__/app.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require('supertest');
+const app = require('../app');
+const connection = require('../db/connection');
+
+afterAll(() => {
+  return connection.end();
```

Comment:
We need to make sure we are seeding out test database. Please add your 'beforeEach' to do this :)